limits.go

```diff
@@ -0,0 +1,52 @@
+package cobrautil
+
+import (
+	"log/slog"
+
+	"github.com/KimMachineGun/automemlimit/memlimit"
+	"github.com/rs/zerolog"
+	slogzerolog "github.com/samber/slog-zerolog/v2"
+	"github.com/spf13/cobra"
+	"go.uber.org/automaxprocs/maxprocs"
+)
+
+// SetLimitsRunE wraps the RunFunc with setup logic for memory limits and maxprocs
+// limits for the go process. It requests 90% of the memory available and respects
+// kubernetes cgroup limits, and requests all of the available CPU quota.
+// NOTE: this assumes that there is already a zerolog instance configured for the process
+// by the time this RunE is invoked.
+func SetLimitsRunE() CobraRunFunc {
```

Comment:
Do we perhaps want to break these out into two different functions with separate concerns?

Call-sites could look like this:

```go
cobrautil.CommandStack(
  SetMemLimitsRunE,
  SetMaxProcsLimitsRunE,
)
```

Reply:
I'm fine with that